python/src/aac/plugins/plugin_manager.py

```diff
@@ -91,7 +93,9 @@ def get_plugin_definitions() -> list[Definition]:
     plugin_definitions = []
     for plugin_definition in plugin_definitions_as_yaml:
         if len(plugin_definition) > 0:
-            plugin_definitions.append(parser.parse(plugin_definition))
+            # Temporary setting until plugins can be updated to provide better contextual information
```

Comment:
We should update the plugin change story's AC to require removing these comments.

Reply:
Done